.github/workflows/plugins.yml

```diff
@@ -16,6 +16,10 @@ concurrency:
 
 jobs:
   aerospike-node-16:
+    strategy:
+      matrix:
+        node-version: [16]
```

Comment:
Seems like node-version is not actually used?

Reply:
Yeah, it's for validation in the script. I'm gonna try to work out a better way of expressing this though.